juju/juju
# Remove view layer from core/network

## Description of change

The view layer ended up inside core/network and by view layer, the CLI
output. This would have locked stepped us into a scenario where it would
make it harder (over time) to change the API without changing the CLI.

This fixes that and also fixes an issue where remove wasn't correctly
showing the warnings when NOT using force.

## QA steps

Ensure that the tests pass, but the following steps can be used:

https://github.com/juju/juju/pull/11183
